interface/package.json

```diff
@@ -17,7 +17,8 @@
   "dependencies": {
     "hclusterjs": "^1.3.1",
     "multtest": "^0.1.7",
-    "ttest": "^1.1.0"
+    "ttest": "^1.1.0",
+    "d3-network": "git+https://github.com/greenelab/d3-network.git"
```

Comment:
:+1:

Reply:
Actually, I have one more question here: Shouldn't this point to the URL for Aaron's repo since it is the one that is in the package.json file of the d3-network package we are now using (and I remember in the previous PR you were saying that this was so because it was the master of the greenelab one).

Reply:
Good question. I did it on purpose in case Aaron changed his mind. Then we will still use our own fork.

Reply:
Ok, good idea :+1: